### DISCUSSION TITLE
Warning:[antd:notification] Static function can not consume context like dynamic theme. Please use 'App' component instead.

### DISCUSSION THREAD

**Discussion by Jarsore** [ant-design/ant-design#50830] [Q&A]
在使用下拉搜索框时遇到的问题，但是之前这么写没遇到过，请问该怎么解决呀？（两个搜索条件框）

**Comment by zombieJ** [comment]
按照提示换成 App 组件即可：

https://ant-design.antgroup.com/components/app-cn#%E4%BD%95%E6%97%B6%E4%BD%BF%E7%94%A8

**Reply by Jarsore** [reply]
在应用顶层中已经包裹App了，感觉很奇怪

**Reply by afc163** [reply]
顶层有一个 App 就够了，可以随处 App.useApp 调用 message 和 notification。

**Reply by Jarsore** [reply]
> 顶层有一个 App 就够了，可以随处 App.useApp 调用 message 和 notification。

我的notification写在onError()里面{notification.error({message:'...'})这种，之前写的就没有这个报错，百思不得其解

**Comment by Jarsore** [comment]
程序正常运行了，不过onSuccess{}跟onError{}都执行了一遍，注释掉onError{}暂时代码可以正常运行；
